web/themes/custom/vote_gov/templates/block/block-content--government-banner.html.twig

```diff
@@ -60,10 +63,11 @@
           <p>
             <strong>{{ content.field_https_heading | field_value }}</strong>
             <br>
-            {% set lock_icon = '(<span class="icon-lock"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image">
+            {% set lock_icon = '(<span class="icon-lock"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-label="lock icon">
+                    <title>lock icon</title>
                     <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z">
                     </path>
-                  </svg></span>)' %}
+                  </svg></span>)' | replace({ 'lock icon' : 'lock icon' | t }) %}
```

Comment:
I'm not sure what you trying to do here. Would it work if you just do it like this?
`</svg></span>)' | t %}`

Reply:
It's replacing the string for the aria-label and title of "lock icon" with it's translation. We don't have those translations yet. I created a ticket for implementing translations for aria-labels and titles (VOTE-1157). I could remove this from the PR and add it to that ticket to implement later?